tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -147,7 +147,8 @@ export class WebGPUBackend extends KernelBackend {
     this.queue = device.queue;
     this.currentCommandEncoder = null;
     this.currentComputePass = null;
-    this.supportTimeQuery = device.features.has('timestamp-query');
+    this.supportTimeQuery =
+        device.features.has('timestamp-query-inside-passes');
```

Comment:
nit: Please add this new ext spec address above the code.

Reply:
This extension is maintained at https://github.com/gpuweb/gpuweb/blob/main/proposals/timestamp-query-inside-passes.md, not written into WebGPU Spec.